.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Build Publish Chainlink'
+
+on:
+  push:
+    branches:
+      - 'release/**'
+
+jobs:
+
+  build-publish-chainlink:
+    # buildjet-2vcpu-ubuntu-1804
+    runs-on: ubuntu-1804
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Install AWS CLI'
+        run: |
+          if [[ -z "$BUILDER" ]]; then
```

Comment:
We don't need this environment variable as long as our referenced environment is restricted to `release/**` only, as it'll only expose secrets to that protected branch that a subset of contributors have access to.

Reply:
If you end up having to duplicate this job across multiple environments/workflows, then you can make use of https://docs.github.com/en/actions/creating-actions/creating-a-composite-action to reuse your referenced steps.

An example of this would be another workflow to test building the container but not testing it, that runs on every PR referencing develop. It would have access to a `develop`-like environment that lets it pull anything needed, but not push anything